geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheServerStats.java

```diff
@@ -591,6 +612,11 @@ public CacheServerStats(StatisticsFactory statisticsFactory, String ownerName, S
     messageBytesBeingReceivedId = stats.nameToId("messageBytesBeingReceived");
     connectionsTimedOutId = stats.nameToId("connectionsTimedOut");
     threadQueueSizeId = stats.nameToId("threadQueueSize");
+    selector0Count = stats.nameToId("selector0Count");
```

Comment:
This only allows for having at most 5 selectors configured. I do not know if a more general solution could be devised.

Reply:
Actually you can configure more - as you can pass desired value - but only 5 you will be able to see in the Stats.

e.g. 

Command-4 : start server --name=server3 --locators=127.0.0.1[10334] --server-port=0 --log-level=DEBUG --classpath=../build/classes/java/main --max-threads=4 **--J=-DBridgeServer.SocketSelectorCount=6**

Status    : PASSED

[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 0
[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 1
[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 2
[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 3
[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 4
[info 2023/01/13 12:04:28.635 CET server3 <main> tid=0x1] Using max-threads and initializing selector 5

I also do not like this but I do not think that it is the right time now to change this feature functionally,
as it has been tested within XXL scope before this Code Review that we are doing now.

So, I would not advise that we are re-designing  this part now - at least not for this release.

Reply:
I do not think that this solution for the stats would have easily passed the Apache community review filter, at least without a proper justification.
I think it is worth investigating if there is a way to have a general solution before just putting this in.

Reply:
As we discussed and agreed yesterday - we will apply the following approach:

-> we need to fix failing CI tests (code or tests - whatever we identify as root cause)
-> we add necessary additional unit/integration tests & documentation update
-> and we then refactor code - where it is to complex - and add functionality to have a general solution for all stats - but this bigger changes we do for the next release